platform/logging/LogRecord.ts

```diff
@@ -0,0 +1,14 @@
+import { LogLevel } from './LogLevel';
+
+/**
+ * Interface describing essential parts of every log message.
```

Comment:
Nitpick: I think we should be more concise with our comments by removing redundant introductory text like "Interface describing".  As an example, this comment could simply be `Essential parts of every log message`, and I don't think we lose any information there since the language itself makes it clear that this is an interface.

It doesn't really matter on any one instance, but when applied to comments throughout a code base, this can help lead to shorter and more to-the-point comments.